edgelet/Cargo.toml

```diff
@@ -15,3 +15,22 @@ members = [
     "management",
     "systemd",
 ]
+
+[patch."https://github.com/Azure/iot-identity-service"]
```

Comment:
How will this interact with the upcoming witch over to using cargo deps instead of git submodules?

Pinging for visibility @gordonwang0

Reply:
That switch already happened. That's why this patch is what it is.

Also, this is temporary. See TODO number 1.

Reply:
Ahhh, gotcha. All good then.